### DISCUSSION TITLE
Synology docker and adding a new device: How to?

### DISCUSSION THREAD

**Discussion by Rayhee** [Koenkk/zigbee2mqtt#6407] [General]
Hi,
I'm running zigbee2mqtt in a docker on my Synology NAS. Everything is working great!. It's easy to connect devices and work with them.
At this moment, i have a new device (a curtain motor). It is not supported yet:

**Received message from unsupported device with Zigbee model 'TS0601' and manufacturer name '_TZE200_5sbebbzs'
Please see: https://www.zigbee2mqtt.io/how_tos/how_to_support_new_devices.html.**
 
I read and follow the instructions from Koen (https://www.zigbee2mqtt.io/how_tos/how_to_support_new_devices.html),
- i can clone the github repository,
- i can not  add the volume to my container: the suggested docker run script is giving problems (-v /opt/zigbee-herdsman-converters:/app/node_modules/zigbee-herdsman-converters)

At this moment, im a little bit lost how to solve this issue on my side.
Is there anyone who is using Zigbee2mqtt on a Synolgy system and is able to tweak the devices.js file? any help is much appreciated

Raymond

**Comment by Rayhee** [comment]
Ticket can be closed: i removed the container and create it again with the addition of the new volume. That was the missing part. My mistake.
After adding the device to the devices.js (i only added the type to the already existing Curtain motor sessions) it works!

**Reply by Korte68** [reply]
Hi, what kind of nas do you have? I have a 712+ but not running...Can you help me please? 
[zigbee2mqtt.html.log](https://github.com/Koenkk/zigbee2mqtt/files/6252521/zigbee2mqtt.html.log)
Thank you
